fusion/cone_pos[] cones
float64 stamp
